app/models/program.rb

```diff
@@ -30,6 +30,10 @@ def scheduled
       where.not(start_time: nil).where.not(room: nil)
     end
 
+    def unscheduled
+      confirmed.where('start_time IS NULL OR room_id IS NULL')
```

Comment:
Please also take https://github.com/openSUSE/osem/pull/1082 into account! Seems like @nishanthvijayan is also working on sth in this direction!